web/packages/shared/components/AuthorizeDeviceWeb/AuthorizeDeviceWeb.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Box, Flex, ButtonPrimary, Text, ButtonLink } from 'design';
+import { Link } from 'react-router-dom';
+
+import cfg from 'teleport/config';
+
+import {
+  DownloadConnect,
+  DownloadLink,
+} from 'shared/components/DownloadConnect/DownloadConnect';
+
+export const DeviceTrustConnectPassthrough = ({
```

Comment:
I added this component to the `shared` package because I will also be exporting the dialog that will be used in Connect to authorize in the final PR. I can be convinced otherwise